server/src/services/job.service.ts

```diff
@@ -233,7 +233,7 @@ export class JobService {
 
       case JobName.STORAGE_TEMPLATE_MIGRATION_SINGLE: {
         if (item.data.source === 'upload') {
-          await this.jobRepository.queue({ name: JobName.GENERATE_JPEG_THUMBNAIL, data: item.data });
+          await this.jobRepository.queue({ name: JobName.GENERATE_THUMBNAIL, data: item.data });
```

Comment:
This should queue `GENERATE_PREVIEW` I believe?

Reply:
Yep, that's right